src/core/js/libraries/jquery.a11y.js

```diff
@@ -1,532 +1,861 @@
-            $("body").a11y_aria_label(true);
-            //ADDS TAB GUARG EVENT HANDLER
-            reattachFocusGuard();
+        function a11y_injectControlElements() {
+            if ($(domSelectors.selected).length === 0) $('body').append($(domInjectElements.selected))
+            if ($(domSelectors.focuser).length === 0)$('body').append($(domInjectElements.focuser))
         }
 
-    };
-
-//TOGGLE ACCESSIBILITY
-    //MAKES CHILDREN ACCESSIBLE OR NOT
-    $.a11y_on = function(isOn, selector) {
-        selector = selector || 'body';
-        isOn = isOn === undefined ? true : isOn;
-        if (isOn === false) {
-            $(selector).attr("aria-hidden", true);
-            $.a11y.options.isOn = false;
-        } else {
-            $(selector).removeAttr("aria-hidden");
-            $.a11y.options.isOn = true;
+        function a11y_removeNotAccessibles() {
+            //STOP ELEMENTS WITH .not-accessible CLASS FROM BEING IN TAB INDEX
+            $(".not-accessible[tabindex='0'], .not-accessible [tabindex='0']").attr({
+                "tabindex": "-1",
+                "aria-hidden": true
+            }).addClass("aria-hidden");
         }
-    };
-
-    $.fn.a11y_on = function(isOn) {
-        isOn = isOn === undefined ? true : isOn;
-        if (isOn) {
-            this.find(hideableElements).filter(tabIndexElementFilter).attr("aria-hidden", "true").attr("tabindex", "-1").addClass("aria-hidden");
-        } else {
-            this.find(hideableElements).filter(tabIndexElementFilter).attr("aria-hidden", "false").removeAttr("tabindex").removeClass("aria-hidden");
+
+        function a11y_disabledAccessibleTabElements() {
+            var accessibleTabElements = $(domSelectors.focusableElementsAccessible);
+            accessibleTabElements.attr({
+                "aria-hidden": "true",
+                "tabindex": "-1"
+            });
         }
-        this.find(tabIndexElements).filter(tabIndexElementFilter).a11y_cntrl(isOn);
-        return this;
-    };
 
+        //TURN ON ACCESSIBILITY FEATURES
+        $.a11y = function(isOn, options) {
+            if ($.a11y.options.isDebug) console.log("$.a11y called", isOn, options )
+            return this;
+        };
 
-//MAKE ACCESSIBLE CONTROLS
+        $.a11y.options = {
+            focusOffsetTop: 0,
+            focusOffsetBottom: 0,
+            animateDuration: 250,
+            OS: "",
+            isTouchDevice: false,
+            isTabbableTextEnabled: false,
+            isUserInputControlEnabled: true,
+            isFocusControlEnabled: true,
+            isFocusLimited: false,
+            isRemoveNotAccessiblesEnabled: true,
+            isAriaLabelFixEnabled: true,
+            isFocusWrapEnabled: true,
+            isScrollDisableEnabled: true,
+            isScrollDisabledOnPopupEnabled: true,
+            isSelectedAlertsEnabled: false,
+            isAlertsEnabled: false,
+            isDebug: false
+        };
```

Comment:
this shows what is on in the library by default, when accessibility._isEnabled: false